academy/xl-run-prod/2-keys.md

```diff
@@ -7,20 +7,61 @@ tag: deep-dive
 
 # Prepare a validator and keys
```

Comment:
This section is interesting and application for all chains, let's say I want to validate Cosmos Hub, these are good practices too. We could maybe link that page in the cosmos hub too.

Reply:
I think that is a good idea and will keep a note to do it once we have the Cosmos Hub content included. 😉